README.md

```diff
@@ -465,6 +465,217 @@ node followerDiff.js
 
 * **PARALLEL_SCANS**: number of databases to scan in parallel (default 4)
 
+## Scan Procedure
+
+In order to scan a set of raft sessions to detect discrepancies
```

Comment:
iirc, the use-case is to detect leader-follower discrepancies, correct? If so, lets mention that

Reply:
I'd say that the *goal* is to detect any discrepancy between members, and the *means* is checking follower-leader discrepancies, due to how technically this is achievable without introducing downtime.

Reply:
I'll add an extra paragraph to explain the general idea of the procedure.